OpenTap.Plugins.PNAX/General/Differential IQ/DIQChannel.cs

```diff
@@ -0,0 +1,83 @@
+﻿// Author: CMontes
+// Copyright:   Copyright 2023 Keysight Technologies
```

Comment:
All the copyrights should probably be updated with 2024:
`// Copyright:   Copyright 2023-2024 Keysight Technologies`

Reply:
Thanks, I updated all files with 2024